base_rest/components/service.py

```diff
@@ -151,7 +151,7 @@ def dispatch(self, method_name, *args, params=None):
         :return:
         """
         method = getattr(self, method_name, object())
-        params = params or {}
+        params = params or request.get_json_data() or {}
```

Comment:
@sbejaoui At this level, we should not use the `request` object. The parameters should be properly passed to the `dispatch` by callers. I've the feeling that the problem is upstream into the call stack.